frontend/src/pages/DetailedCourse/DetailedCourseContainer.tsx

```diff
@@ -1,4 +1,5 @@
-import React from 'react';
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
Is it necessary to use eslint-disable here?

Reply:
Removed